components/script/dom/window.rs

```diff
@@ -1676,7 +1659,7 @@ impl Window {
         // objects removed from the tree that haven't been collected
         // yet). There should not be any such DOM nodes with layout
         // data, but if there are, then when they are dropped, they
-        // will attempt to send a message to the closed layout thread.
+        // will attempt to send a message to layout.
         // This causes memory safety issues, because the DOM node uses
         // the layout channel from its window, and the window has
```

Comment:
Is this comment still relevant?

Reply:
I think this is still relevant, because DOM nodes still access the layout channel by first talking to the `Window`. I hope that we can clean this up and make this impossible in a later cleanup, but it's all very delicate and I don't want to rework too much in one change.